tests/toolbox_image.sh

```diff
@@ -129,6 +130,14 @@ test_execute_rsync()
     ( chroot "${rootfs_dir}" /usr/bin/rsync --version 1> /dev/null && return 0 ) || return 1
 }
 
+test_system_update_entrypoint()
+{
+    if [ ! -x "${rootfs_dir}${SYSTEM_UPDATE_ENTRYPOINT}" ]; then
+        return 1
+    fi
+    return 0
```

Comment:
just return the result of the test
where it may be more readable to use the binary as 'test' rather then the symlink [ (yes that's the name of the binary ;) )
```sh
return test -x "${rootfs_dir}${SYSTEM_UPDATE_ENTRYPOINT}"
```
doesn't even need to be negated :)